hedera-node/hedera-consensus-service-impl/src/main/java/com/hedera/node/app/service/consensus/impl/handlers/ConsensusCreateTopicHandler.java

```diff
@@ -64,6 +64,7 @@ public void preHandle(@NonNull final PreHandleContext context) throws PreCheckEx
         // The transaction cannot set the admin key unless the transaction was signed by that key
         if (op.hasAdminKey()) {
             context.requireKeyOrThrow(op.adminKey(), BAD_ENCODING);
+            //  context.requireKeyOrThrow(op.adminKey(), INVALID_ADMIN_KEY); ref #7770
```

Comment:
Why would we want to change this to `INVALID_ADMIN_KEY` if the mono service already returns `BAD_ENCODING`?